src/components/DisplayCard/DisplayCard.tsx

```diff
@@ -1,14 +1,104 @@
 import { capitaliseFirstLetter } from "../../utils/stringFunctions";
 
+import style from "./DisplayCard.module.css";
+
+type EntityStats = {
+  base_stat: number;
+  stat: {
+    name: string;
+    url: string;
+  };
+};
+
 export type DisplayCardProps = {
   key: number;
   name: string;
+  spriteFront: string;
+  stats: EntityStats[];
+  type: string;
+};
+
+interface backgroundColourType {
+  [key: string]: {
+    color: string;
+  };
+}
+
+const backgroundColor: backgroundColourType = {
```

Comment:
I think the only comment i really have here is that some of these types, and data sets could be pulled out of here, but probably until we get a better idea of the overall structure of the application there isn't a good place yet to home them.

Maybe something to just keep in the back of the minds to clean this up at a later point once we have a better place for them.